### PR TITLE
Installing qutip in default docker image

### PR DESCRIPTION
**Pull request by simonwa7**

### CODE REVIEW

File: docker/z-quantum-default/0.0.4/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+                    cffi>=1.14.0 \
+                    scipy==1.4.1 \
+                    lea==3.2.0 \
+                    numpy==1.18.1 \
+                    werkzeug==0.14.1 \
+                    flask==1.1.1 \
+                    pyyaml==5.1 \
+                    quantum-grove>=1.0.0
+
+RUN python3 -m pip install qutip
```
Comment (mstechly) [inline docker/z-quantum-default/0.0.4/Dockerfile:74]: Why as a separate command instead of spending it to the previous one?

Comment (simonwa7) [reply docker/z-quantum-default/0.0.4/Dockerfile:74]: There were some issues popping up during the `build` process when it was part of the previous command.

Comment (mstechly) [reply docker/z-quantum-default/0.0.4/Dockerfile:74]: And why no version is specified for qutip?

Comment (simonwa7) [reply docker/z-quantum-default/0.0.4/Dockerfile]: Addressed here: 33c34cf

**Pull request merged**